.github/workflows/test-blackduck-flow.yml

```diff
@@ -77,4 +79,4 @@ jobs:
           blackduck_scan_full: true
           blackduck_install_directory: /Users/kishori/synopsys-bridge
           blackduck_scan_failure_severities: "[\"BLOCKER\", \"CRITICAL\", \"TRIVIAL\"]"
-#          bridge_download_url: ${{ env.MAC_BRIDGE_URL }}
+          configure_from_repo: true
```

Comment:
add configure_from_repo in https://sig-confluence.internal.synopsys.com/pages/viewpage.action?pageId=835892736